frontend/app/controllers/container_profiles_controller.rb

```diff
@@ -1,13 +1,20 @@
 class ContainerProfilesController < ApplicationController
 
-  set_access_control  "view_repository" => [:show, :index, :typeahead],
-                      "update_container_profile_record" => [:new, :edit, :create, :update, :delete]
+  set_access_control  "view_repository" => [:show, :index, :typeahead, :defaults],
```

Comment:
@cpapazoglou `:defaults` should be moved from `"view_repository"` to `"manage_repository"`. See others like [accessions](https://github.com/archivesspace/archivesspace/blob/master/frontend/app/controllers/accessions_controller.rb#L8), [digital objects](https://github.com/archivesspace/archivesspace/blob/master/frontend/app/controllers/digital_objects_controller.rb#L9), etc.